lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
> ...m is still AST_SymbolRef though?

Not necessarily. It could be `foo(x).prop[x - y]`.

> So I guess you can do ...[0, 1]?
> If so, then it should read AST_Node instead 

Correct. It's an arbitrary expression resulting in an array or iterable.